### PR TITLE
feat(pci.k8s.add): add update policy selection

### PR DESCRIPTION
**Pull request by y4nnL**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-329
| License          | BSD 3-Clause

- [X] Try to keep pull requests small so they can be easily reviewed.
- [X] Commits are signed-off
- [X] Only FR translations have been updated
- [X] Branch is up-to-date with target branch
- [X] Lint has passed locally
- [X] Standalone app was ran and tested locally
- [X] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/update-policy-picker.constants.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// English only
+export const DOCUMENTATION_LINK =
+  'https://help.ovhcloud.com/csm/en-public-cloud-kubernetes-change-security-update?id=kb_article_view&sysparm_article=KB0049649';
```
Comment (brofik) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/update-policy-picker.constants.js:3]: The documentation link should be specific to the subsidiary of the user's account.

Comment (y4nnL) [reply packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/update-policy-picker.constants.js:3]: The documentation only exists in english, hence the comment on line 1

**Review by sachinrameshn — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by darsene — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import angularTranslate from 'angular-translate';
+import uiKit from '@ovh-ux/ui-kit';
+
+import component from './update-policy-picker.component';
+
+const moduleName = 'ovhManagerPciKubeUpdatePolicyPicker';
+
+angular
+  .module(moduleName, [angularTranslate, uiKit])
+  .component('kubeUpdatePolicyPicker', component)
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/index.js:11]:
```suggestion
  .component('pciProjectKubeUpdatePolicyPicker', component)
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by brofik — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/index.js (hunk 2 above)
Comment (y4nnL) [reply packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/index.js:11]: Mmm I disagree @anooparveti 
The plan is to make kubernetes a µapp so let's start right now by having the correct name for new components right ?

Comment (anooparveti) [reply packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/index.js:11]: Nope, i dont agree. As long as we are in monolith, we have to follow it's conventions.

**Review by anooparveti — APPROVED** [review]

**Review by brofik — APPROVED** [review]

**Pull request closed without merging**